client.go

```diff
@@ -30,6 +30,8 @@ type Config struct {
 	BasicAuth *url.Userinfo
 	// Client provides an optional HTTP client, otherwise a default will be used.
 	Client *http.Client
+	// OrgID provides an optional organization ID, ignored when using APIKey, BasicAuth defaults to last used org
+	OrgID string
```

Comment:
I suggest this to be an `int64` instead to mimic the behavior in Grafana. If it's greater than zero it's supposed to be set.

Reply:
Updated to int64, thanks